app/utils/file.test.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getExtensionFromContentDisposition} from 'app/utils/file';
+
+describe('getExtensionFromContentDisposition', () => {
+    it('should return the extracted the extension', () => {
+        const exts = [
+            'png',
+            'PNG',
+            'gif',
+            'GIF',
+            'bmp',
+            'BMP',
+            'jpg',
+            'JPG',
+            'jpeg',
+            'JPEG',
```

Comment:
Is there a possibility of having invalid extension type? If yes, then worth adding a test case for it.